core/core-keybindings.el

```diff
@@ -42,27 +99,66 @@ gui, translate to [C-i]. Otherwise, [9] (TAB)."
 ;;     [C-m] [?\C-m]))
 ;; (define-key key-translation-map [?\C-m] 'spacemacs/translate-C-m)
 
-(defun spacemacs/declare-prefix (prefix name &optional long-name)
+(defun spacemacs/declare-prefix (prefix name &optional long-name redeclare)
```

Comment:
@syl20bnr  redeclare used for whitelisting re-bindings. It ensures that a function user knows what will be replaced. 
We can whitelist replacement that are identical by default, but I think that detecting them can help in fighting copypasta errors.